src/component/mxgraph/BpmnGraph.ts

```diff
@@ -64,10 +64,10 @@ export class BpmnGraph extends mxgraph.mxGraph {
    * @experimental subject to change, may move to a subclass of `mxGraphModel`
    * @alpha
    */
-  batchUpdate(fn: () => void): void {
+  batchUpdate(function_: () => void): void {
```

Comment:
**suggestion:** let's fine a better name without the `_` postfix, as it is weird.

Reply:
It's not possible to rename to `function` because it's a reserved word.